examples/webgl_materials_texture3d_volume1.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>three.js webgl - volume rendering example</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+		body {
+			font-family: Monospace;
+			background-color: #000;
+			color: #fff;
+			margin: 0px;
+			overflow: hidden;
+		}
+		#info {
+			color: #fff;
+			position: absolute;
+			top: 10px;
+			width: 100%;
+			text-align: center;
+			z-index: 5;
+			display:block;
+		}
+		.dg {
+			z-index: 10 !important;
+		}
+		#info a, .button { color: #f00; font-weight: bold; text-decoration: underline; cursor: pointer }
+		#inset  {
+			width: 150px;
+			height: 150px;
+			background-color: transparent; /* or transparent; will show through only if renderer alpha: true */
+			border: none; /* or none; */
+			margin: 0;
+			padding: 0px;
+			position: absolute;
+			left: 20px;
+			bottom: 20px;
+			z-index: 100;
+		}
+	</style>
+</head>
+
+<body>
+	<div id="info">
+		<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> -
+		Float volume render test (mip / isosurface)
+	</div>
+	<div id="inset"></div>
+
+	<script src="../build/three.js"></script>
+
+	<script src="js/controls/TrackballControls.js"></script>
+
+	<script src="js/Volume.js"></script>
+	<script src="js/VolumeSlice.js"></script>
+	<script src="js/loaders/NRRDLoader.js"></script>
+	<script src="js/shaders/VolumeShader.js"></script>
+
+	<script src="js/Detector.js"></script>
+	<script src="js/libs/stats.min.js"></script>
+	<script src="js/libs/gunzip.min.js"></script>
+	<script src="js/libs/dat.gui.min.js"></script>
+
+	<script>
+
+		if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+		var container,
+			stats,
+			camera,
+			controls,
+			scene,
+			renderer,
+			gui,
+			container2,
+			renderer2,
+			camera2,
+			axes2,
+			scene2;
+
+		init();
+		animate();
+
+		function init() {
+
+			// The volume renderer does not work very well with perspective yet
```

Comment:
Is this a limitation of the existing Volume.js code or a limitation of the newly added Volume Shader?

Reply:
Good question. I am not entirely sure, to be honest. It could be a limitation of the shader technique, but it can also be due to subtle texture coordinate alignment stuff. I *think* that I recall having the same technique working with a perspective view in the VisPy project. I'd need to dive into this at some point.

Reply:
I should add that the rendering does work with a perspective camera, but it gets "wobbly" at the edges at certain angles.